src/backend/Backend_intf.ml

```diff
@@ -17,11 +17,9 @@ module type S = sig
       according to the conventions of a given format.
   *)
 
-  type t
   (** The type of proofs. *)
+  type t
```

Comment:
I personally prefer doc comment following the declarations (in signatures), is it configurable in ocamlformat ?